Emby.Server.Implementations/HttpServer/HttpListenerHost.cs

```diff
@@ -58,6 +59,7 @@ public class HttpListenerHost : IHttpServer, IDisposable
             _logger = logger;
             _config = config;
             _defaultRedirectPath = configuration["HttpListenerHost:DefaultRedirectPath"];
+            _baseUrlPrefix = _config.Configuration.BaseUrl;
```

Comment:
This is redundant now.

Reply:
Perhaps, but cleaner to just have it in its own variable, no?

Reply:
Again, `_config.Configuration.BaseUrl` *could* get updated and `_baseUrlPrefix` won't change, this could lead to subtle bugs, unless this is intended.

Reply:
Right, we *do* want to avoid that. It should remain the exact same until the server is restarted.